libs/qCC_db/ccPointCloudLOD.cpp

```diff
@@ -27,6 +27,8 @@
 //! Thread for background computation
 class ccPointCloudLODThread : public QThread
 {
+	Q_OBJECT
```

Comment:
There are several other reasons listed [here](https://github.com/KDE/clazy/blob/master/src/checks/level1/README-missing-qobject-macro.md).

If it's not included, then using qobject_cast won't work for example.

My view is that it's cleaner to always include it. That way when you do need to use one of those other features you won't be spending time trying to figure out why it doesn't work.